library/opusencoder/src/main/java/im/vector/opusencoder/configuration/SampleRate.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.opusencoder.configuration
+
+/**
+ * Sampling rate of the input signal in Hz.
+ */
+sealed class SampleRate private constructor(val value: Int) {
+    object _8khz : SampleRate(8000)
```

Comment:
Starting with `_` prefix is not allowed by lint.

Reply:
I honestly can't find a better 

> Code Quality Checks failed, did you check it ? :)

Yes, it's related to this: https://github.com/vector-im/element-android/pull/6208#discussion_r891251197

Reply:
In the end I went with a `Rate` prefix. It's a bit weird, but it should fix the issue.